File: content/lessons/01-getting-started/01-hello-opmode/Solution.java
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

@TeleOp(name = "Hello OpMode")
public class StudentCode extends LinearOpMode {
    @Override
    public void runOpMode() {
        telemetry.addData("Message", "Hello, FTC!");
        telemetry.update();
        
        waitForStart();
        
        telemetry.addData("Status", "Running");
        telemetry.update();
    }
}

File: content/lessons/01-getting-started/01-hello-opmode/Solution.kt
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
import com.qualcomm.robotcore.eventloop.opmode.TeleOp

@TeleOp(name = "Hello OpMode")
class StudentCode : LinearOpMode() {
    override fun runOpMode() {
        telemetry.addData("Message", "Hello, FTC!")
        telemetry.update()

        waitForStart()

        telemetry.addData("Status", "Running")
        telemetry.update()
    }
}
